ios/appservice/appservice.go

```diff
@@ -76,7 +157,40 @@ func buildAppLaunchPayload(deviceId string, bundleId string, args []interface{},
 	})
 }
 
-func buildCoreDevicePayload(deviceId string, feature string, input map[string]interface{}) map[string]interface{} {
+func buildListProcessesPayload(deviceId string) map[string]interface{} {
+	return buildRequest(deviceId, "com.apple.coredevice.feature.listprocesses", nil)
+}
+
+func buildRebootPayload(deviceId string) map[string]interface{} {
+	return buildRequest(deviceId, "com.apple.coredevice.feature.rebootdevice", map[string]interface{}{
+		"rebootStyle": map[string]interface{}{
```

Comment:
`devicectl` has a `--style` argument to decide what kind of reboot to execute. Available options are `--style userspace` and `--style full`. I understand that this payload initiates a full reboot of the kernel.